librz/util/str.c

```diff
@@ -1782,6 +1782,20 @@ RZ_API bool rz_str_is_ascii(const char *str) {
 	return true;
 }
 
+RZ_API bool rz_str_is_utf8(const char *str) {
```

Comment:
Doxygen please. Also `RZ_NONNULL` and `rz_return_val_if_fail(str, false)`.

Reply:
Ok ff35d9121dd42c747e55cafef2be9ebae7ce02cc